Webpack error when attempting to add component to .Net Core Angular2 template

I am trying to add IgGridComponent to an Angular2 project created from a VS2015 Angular2 template that uses webpack. Details on that template [here](http://blog.stevensanderson.com/2016/10/04/angular2-template-for-visual-studio/)

I have installed the igniteui-angular2 package (1.1.2) but webpack complains with the following:

> ERROR in ./~igniteui-angular2/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./igniteui in C:\project_path\node_modules\igniteui-angular2 at ./~igniteui-angular2/index.js 21:0-21

New to Angular2 so any help would be appreciated.

Thanks

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
     "watch": "node node_modules/typescript/bin/tsc -w",
     "build": "node node_modules/typescript/bin/tsc",
     "bundle": "node node_modules/typescript/bin/tsc -d && npm run prepare-dist",
-    "prepare-dist": "cp src/igniteui.angular2.js dist/npm/index.js  && cp src/igniteui.angular2.d.ts dist/npm/index.d.ts && cp src/igniteui.angular2.ts dist/npm/igniteui.angular2.ts && cp src/igniteui.ts dist/npm/ && cp src/jquery.d.ts dist/npm/",
+    "prepare-dist": "cp src/igniteui.angular2.js dist/npm/index.js  && cp src/igniteui.angular2.d.ts dist/npm/index.d.ts && cp src/igniteui.angular2.ts dist/npm/igniteui.angular2.ts && cp src/igniteui.ts dist/npm/ && cp src/igniteui.js dist/npm/ && cp src/igniteui.js.map dist/npm/ && cp src/jquery.d.ts dist/npm/",
 
     "pretest": "npm run build",
     "test": "karma start tests/karma.conf.js && npm run remap-istanbul",

